src/pages/DetailsPage.js

```diff
@@ -62,6 +71,7 @@ const defaultProps = {
     session: {
         email: null,
     },
```

Comment:
Do we still need `email` (session)?

Reply:
There is no usage of it anymore, I think we can remove it

Reply:
@s77rt updated!